dist/znk.infra.js

```diff
@@ -3121,6 +3309,89 @@ angular.module('znk.infra.popUp').run(['$templateCache', function($templateCache
     "");
 }]);
 
+'use strict';
+
+(function (angular) {
+    angular.module('znk.infra.presence').provider('PresenceService', function () {
+
+        var AuthSrvName;
+
+        this.setAuthServiceName = function (authServiceName) {
+            AuthSrvName = authServiceName;
+        };
+
+        this.$get = [
+            '$log', '$injector', 'ENV', '$rootScope',
+            function ($log, $injector, ENV, $rootScope) {
+                var PresenceService = {};
+                var authService = $injector.get(AuthSrvName);
+                var rootRef = new Firebase(ENV.fbDataEndPoint, ENV.firebaseAppScopeName);
```

Comment:
we cannot use new Firebase in more than one place in infra. We should a function for it in firebaseStorageAdapter